servers/rendering/rendering_device.h

```diff
@@ -368,7 +368,7 @@ class RenderingDevice : public RenderingDeviceCommons {
 	// used for the render pipelines.
 
 	struct AttachmentFormat {
-		enum { UNUSED_ATTACHMENT = 0xFFFFFFFF };
+		enum : uint32_t { UNUSED_ATTACHMENT = 0xFFFFFFFF };
```

Comment:
Let's fix the style too.
```suggestion
		enum : uint32_t {
			UNUSED_ATTACHMENT = 0xFFFFFFFF
		};
```